### PR TITLE
Updated testcases

### PR DESCRIPTION
**Pull request by ashish123mahajan**
Updated MemoryComsumptionTest.java and GenericTestUtils for providing runtime mock service.

### CODE REVIEW

**Comment by ashish123mahajan** [comment]
Updated MemoryComsumptionTest.java and GenericTestUtils for providing runtime mock service.

File: a2d2-api/src/test/java/io/elimu/a2d2/processtest/GenericServiceTest.java [hunk 1]
```diff
@@ -63,7 +63,7 @@ public static void setUpStatic() {
 	}
 	
 	@Before
-	public void setUp() throws Exception {
+	public void setUp() throws Exception {
```
Comment (hankwallace) [inline a2d2-api/src/test/java/io/elimu/a2d2/processtest/GenericServiceTest.java:66]: It's a minor thing, but you should make sure trailing spaces are not added.

File: a2d2-api/src/test/java/io/elimu/a2d2/processtest/GenericServiceTest.java [hunk 2]
```diff
@@ -73,82 +73,14 @@ public void setUp() throws Exception {
...
-	  		if (in != null) {
-	  			in.close();
-	  		}
-	  	}
-	}
+
 	
 	private void deployManualKjar() throws IOException {
-		deployKjar("generic-example-service.dsl", "1");
+		GenericTestUtils.deployKjar("generic-example-service.dsl", "1");
```
Comment (hankwallace) [inline a2d2-api/src/test/java/io/elimu/a2d2/processtest/GenericServiceTest.java:79]: @kevinliuelimu do we have a best practice for indentation - like the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s4.6-whitespace)? I see tabs in some source files and spaces in another. Let's just decide to be consistent.

File: a2d2-api/src/test/java/io/elimu/a2d2/processtest/GenericServiceTest.java [hunk 3]
```diff
@@ -208,7 +140,7 @@ public void testKieServiceAuthWithDatabase() throws Exception {
 		//Assert.assertNotNull(request.getUser());//user set from Authorization header
 		//Assert.assertEquals("user", request.getUser());
 		Assert.assertNotNull(response);
-		Assert.assertNotNull(response.getBody());
+		Assert.assertNotNull(response.getBody());
```
Comment (hankwallace) [inline a2d2-api/src/test/java/io/elimu/a2d2/processtest/GenericServiceTest.java:143]: Please remove trailing whitespace

**Review by hankwallace — CHANGES_REQUESTED** [review]
A few small changes requested. Please get Kevin's or Mariano's feedback before merging.

**Review by marianbuenosayres — APPROVED** [review]

File: a2d2-api/src/test/java/io/elimu/a2d2/processtest/GenericServiceTest.java (hunk 1 above)
Comment (ashish123mahajan) [reply a2d2-api/src/test/java/io/elimu/a2d2/processtest/GenericServiceTest.java]: I have updated as per discussion

**Review by hankwallace — APPROVED** [review]

File: a2d2-api/src/test/java/io/elimu/a2d2/processtest/GenericServiceTest.java (hunk 2 above)
Comment (kevinliuelimu) [reply a2d2-api/src/test/java/io/elimu/a2d2/processtest/GenericServiceTest.java:79]: We should use tabs for indentation.

**Pull request merged**